Assets/Scripts/Inventory/Custom/Hands.cs

```diff
@@ -1,146 +1,148 @@
 ﻿using System.Linq;
 using Mirror;
 using UnityEngine;
-using UnityEngine.EventSystems;
-
-/**
- * The hand system is an interaction system which connects to the inventory.
- * There is a hands container, that the interaction system connects to, that defines what items the player is currently holding.
- */
-[RequireComponent(typeof(Inventory))]
-public class Hands : Interaction, Tool
+
+namespace Inventory.Custom
```

Comment:
I'm not sure what the `Custom` part of this means

Reply:
It's just the folder its in. All the namespace additions are here only because I was getting conflicts between class names. I did not change any code associated with the new namespaces that I added (other than auto formatting them).